src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
+    Setting the **Default Value** to 1 ensures the Wizard always starts at step 1.
 
+1. From the Toolbox, drag two Button widgets onto your screen and set their names to **Previous** and **Next**.
 
-## Input Parameters
+1. Define the behaviour for each of the navigation buttons. 
 
-### Wizard
+      ![](images/wizard-image-15.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Orientation  |  Set the orientation. | Orientation Identifier | False | Entities.Orientation.Horizontal |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
+1. Create an expression that defines the status of each of the Wizard Items (current step, step already completed, step yet to be completed), depending on what stage the user is at.
 
-### Wizard Item
-
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Step  | Set the step. | Step Identifier | True | none |
-| UseTopLabel  |  If true, label is placed above the icon. |  Boolean | False | False |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
-  
-## Layout and Classes
+    ![](images/wizard-image-16.png)
 
-![](images/wizard-image-3.png?width=750)
+1. Create a condition for both the **Previous** and **Next** buttons. This ensures that the user can never goes above or below the actual number of steps in the Wizard.
 
-## CSS Selectors
+    ![](images/wizard-image-17.png)
 
-| **Element** |  **CSS Class** |  **Description**  |
-| ---|---|---
-| .wizard |  .wizard.wizard-vertical|  When the wizard orientation is vertical  |
-| .wizard-item |  .wizard-item.active|  Defines the current active step  |
-| .wizard-item |  .wizard-item.past|  Defines the previous step  |
-| .wizard-item |  .wizard-item.next|  Defines the next step  |
+**Add Content to your Wizard**
 
-## Advanced Use Case
+1. Create a condition to control what is displayed, depending on what step is active.
+    
+    ![](images/wizard-image-19.png)
 
-### Use Wizard with ListRecords
+1. Add the relevant content to each for each of the steps.
 
-1. Drag the Wizard Pattern into the screen.
+     The following example shows the Shipping Details step that contains text and input boxes.
 
-1. In the Content placeholder, drag a ListRecords widget.
+      ![](images/wizard-image-18.png)
 
-1. Set the Line Separator parameter of the ListRecords to None.
+1. Create variables for each of the elements in the step container. 
+    
+    ![](images/wizard-image-20.png)
 
-1. In the ListRecords widget, drag a WizardItem Pattern.
+**Create a Screen Action**
 
-1. In the WizardItem Pattern, use expressions to display the database content you need.
+To ensure that all of the information the user enters is passed from step to step, you must create a screen action for the **Previous** and **Next** navigation buttons.
```

Comment:
Screen Action (link to documentation)? And maybe bold, not sure about the style guide rules?

Reply:
When can I find the documentation about screen actions?

Reply:
Is this the correct link? https://success.outsystems.com/Documentation/11/Reference/OutSystems_Language/Logic/Implementing_Logic/Logic_Elements/Screen_Action